distribution/docker/Dockerfile

```diff
@@ -90,6 +105,12 @@ RUN mkdir /opt/druid/var /opt/shared \
  && chown druid:druid /opt/druid/var /opt/shared \
  && chmod 775 /opt/druid/var /opt/shared
 
+# Install iproute2 library if busybox not installed
+RUN if [ "$BASE_IMAGE" != "gcr.io/distroless/java$JDK_VERSION-debian11" ]; then \
+     apt update \
+     && apt install -y iproute2; \
```

Comment:
I would first check if `iproute2` and `apt` are available before running this.
Do nothing if `iproute2` is available. If not, fail if `apt` is not available.

Reply:
can we explain why we need to install iproute2 in the first place?

Reply:
again, if this is necessary, I would suggest instead to check if iproute2 is available already and only install it if it is not.

Reply:
`ip` command is not present. This causes for all services, the config property of host ip addresses not set ([Here](https://github.com/confluentinc/druid/blob/25.0.0-confluent/distribution/docker/druid.sh#L140)) and they are not able to communicate with each other. 
I'll update the conditionals and add a more detailed comment.

Reply:
makes sense, let's add a comment to explain. We can check if ip command exists and install if not enabled.